src/lib/axl-explorations/src/components/games/LetterHuntGameCore.tsx

```
@@ -765,8 +765,8 @@ export function LetterHuntGameCore({
 
   const Container: any = useContainer === 'card' ? Card : 'div';
   const containerClass = useContainer === 'card'
-    ? `flex-1 p-4 sm:p-0.5 md:p-1 lg:p-2 bg-white/95 backdrop-blur-sm shadow-floating overflow-y-auto flex flex-col relative ${className}`
-    : `flex-1 p-4 sm:p-0.5 md:p-1 lg:p-2 overflow-y-auto flex flex-col relative ${className}`;
+    ? `flex-1 p-4 sm:p-0.5 md:p-1 lg:p-2 bg-white/95 backdrop-blur-sm shadow-floating overflow-hidden flex flex-col relative ${className}`
```

ESLint

[error] Insert `··`

(prettier/prettier)